### PR TITLE
DEV-642

### PR DESCRIPTION
**Pull request by tranlyvu**

### CODE REVIEW

File: adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class DeterLabProjectEntityTest {
+    private DeterLabProjectEntity deterLabUserEntity;
+
+    @Before
+    public void setup(){
+        deterLabUserEntity=new DeterLabProjectEntity();
+    }
+
+    @Test
+    public void getterAndSetterTest(){
```
Comment (ChrisZhong) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: Please do not combine all tests into one test case

File: adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Test
+    public void testGetId(){
+        final String expected = RandomStringUtils.randomAlphabetic(20);
+        deterLabUserEntity.setId(expected);
+        String actual=deterLabUserEntity.getId();
+
+        assertThat(actual).isNotNull();
+        assertEquals(expected,actual);
```
Comment (ChrisZhong) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: Please use AssertJ syntax

File: adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Test
+    public void testGetNclTeamId(){
+        final String expected = RandomStringUtils.randomAlphabetic(20);
+        deterLabUserEntity.setNclTeamId(expected);
+        String actual=deterLabUserEntity.getNclTeamId();
+
+        assertThat(actual).isNotNull();
+        assertEquals(expected,actual);
```
Comment (ChrisZhong) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: Please use AssertJ syntax

File: adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Test
+    public void testGetDeterProjectId(){
+        final String expected = RandomStringUtils.randomAlphabetic(20);
+        deterLabUserEntity.setDeterProjectId(expected);
+        String actual=deterLabUserEntity.getDeterProjectId();
+
+        assertThat(actual).isNotNull();
+        assertEquals(expected,actual);
```
Comment (ChrisZhong) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: Please use AssertJ syntax

File: adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java (hunk 2 above)
Comment (DCSTLVU) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: I did!  But why do we have to use AssertJ instead?

File: adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+/**
+ * Created by Tran Ly Vu on 10/3/2016.
+ */
+
+public class DeterLabProjectEntityTest {
+    private DeterLabProjectEntity deterLabUserEntity;
+
+    @Before
+    public void setup(){
+        deterLabUserEntity=new DeterLabProjectEntity();
```
Comment (ChrisZhong) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: do not need a `before` method. simply declare and initialize the entity in each test method

File: adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Test
+    public void testGetId(){
```
Comment (ChrisZhong) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: there should be two tests for `getId`. 1) ensure that the initial value is correct 2) test that it returns the correct value after calling `setId`
in addition, there should be a test for the `setId` as well

File: adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java (hunk 2 above)
Comment (ChrisZhong) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: assertEquals is not AssertJ syntax

Comment (tranlyvu) [inline adapter-deterlab/src/test/java/sg/ncl/adapter/deterlab/dtos/entities/DeterLabProjectEntityTest.java]: agreed! its Junit syntax, my question is why don't we use Junit instead?

**Pull request merged**